src/main/resources/db/migration/V1_56__action_plan_meeting_method.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TYPE meeting_method_code AS ENUM ('PHONE_CALL','VIDEO_CALL', 'IN_PERSON_MEETING_OFFICE', 'IN_PERSON_MEETING_OTHER');
+
+create table meeting_location(
+    id uuid not null,
+--     TODO: Waiting for ming to confirm if we should use address field here
+    address text not null,
+    city VARCHAR(35) not null,
+    post_code VARCHAR(8) not null,
```

Comment:
there's a place which is 58 characters in long form, not counting typos: https://en.wikipedia.org/wiki/Llanfairpwllgwyngyll ;)

we can technically fit postcode into 8 characters, after sanitisation and stripping spaces even 7

however, we can not sanitise address/city as efficiently and we _could_ allow for user error, weird formatting, copy/paste errors

however, since `text` and `varchar` in PostgreSQL have the same overhead under 126 bytes:
> The storage requirement for a short string (up to 126 bytes) is 1 byte plus the actual string

... what do you think about using `text` everywhere?

Reply:
To confess I copied the sql fields that were used in community-api as I thought it would be a good reference

Reply:
🙀

Reply:
no longer relevant because we are using address field for everything